skbio/io/tests/test_fasta.py

```diff
@@ -412,7 +428,7 @@ def test_fasta_to_generator_valid_files(self):
                 self.assertEqual(len(obs), len(exp))
                 for o, e in zip(obs, exp):
                     e = e.copy()
-                    del e.positional_metadata
+                    e.positional_metadata.drop('quality', 1, inplace=True)
```

Comment:
If there's lowercase data in `positional_metadata` (ie 'introns' column) we need it retained.